### DISCUSSION TITLE
gitlab danger ci on jenkins

### DISCUSSION THREAD

**Discussion by matheus-gsilva-ifood** [danger/danger-js#1149] [Q&A]
Hello!
Ive been trying to run danger with jenkins and gitlab, but I haven`t being having much luck.

I`ve tried to add 

```
        CI_MERGE_REQUEST_IID=123123 CI_PROJECT_PATH=test123 yarn danger ci
```

As a fallback to check if that would work but I didn`t unfortunately, I keep getting:

`Skipping Danger due to this run not executing on a PR.`

**Comment by orta** [comment]
One of these his not getting set: https://github.com/danger/danger-js/blob/main/source/ci_source/providers/Jenkins.ts#L53-L63

**Reply by matheus-gsilva-ifood** [reply]
That was the problem, now i'm almost there. It is targeting the right URL but somehow missing it when it tries to do a HTTP query:

```
  response: Response {
    size: 0,
    timeout: 0,
    [Symbol(Body internals)]: { body: [PassThrough], disturbed: true, error: null },
    [Symbol(Response internals)]: {
      url: 'https://MY_GITLAB_HOST/api/v4/projects/path%2Fto%2Fmy%2Fcproject%2F-/merge_requests/3/changes',
      status: 404,
      statusText: 'Not Found',
      headers: [Headers],
      counter: 0
    }
  },
  description: '404 Project Not Found'
}
```

What else am I missing?
